styles.css

```diff
@@ -421,10 +403,174 @@ body {
   padding-top: 10px;
 }
 
-.navigation ul {
+.navigation ul,
+#modalContainers {
   display: none;
 }
 
+.alert-Msg {
+  font-size: 12px;
+  font-weight: 600;
+  color: #fff;
+  padding-top: 12px;
+}
+
+/* Modal Design */
+#modalContainers {
+  display: flex;
+  align-content: center;
+  background-color: rgba(100, 104, 140, 0.499);
+  font-family: 'Poppins', sans-serif;
+  list-style: none;
+  height: 100%;
+  position: fixed;
+  top: 0;
+  left: 0;
+  backdrop-filter: blur(6px);
+  padding: 40px 14px 14px 14px;
+  margin-bottom: 50rem;
+}
+
+.modalNavLi a {
+  font-size: 35px;
+  text-decoration: none;
+  color: #67798e;
+}
+
+#modalContainer a {
+  text-decoration: none;
+}
+
+.modalNav {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  list-style: none;
+  justify-content: space-between;
+  padding-bottom: 12px;
+}
+
+.modalHeader {
+  padding: 20px 24px 4px 20px;
+  background-color: #fff;
+  width: 100%;
+  margin: 0 auto;
+  top: 0;
+  right: 0;
+  border-radius: 1rem;
+  overflow: hidden;
+}
+
+#close-btn {
+  font-size: 2rem;
+  cursor: pointer;
+  top: 4%;
+  left: 88%;
+  right: 0;
+}
+
+.modalHeader h3 {
+  font-size: 32px;
+  font-weight: 700;
+}
+
+#canopy {
+  gap: 30px;
+}
+
+.canopy {
+  display: flex;
+  flex-direction: row;
+  gap: 3px;
+  font-size: 13px;
+  font-weight: 600;
+  color: #7a869a;
+  font-family: 'Poppins', sans-serif;
+  padding-bottom: 0.5rem;
+}
+
+.modalFooter li {
+  border-radius: 10px;
+  background-color: #ebebff;
+  padding: 6px 12px 6px 12px;
+  font-size: 12px;
+}
+
+.canopy li:nth-child(1) {
+  font-size: 13px;
+  font-weight: 600;
+  color: #344563;
+  list-style-type: none;
+}
+
+.modalImg {
+  padding-top: 10px;
+}
+
+.modalFooter {
+  padding-bottom: 2px;
+  overflow: hidden;
+}
+
+.modalFooter p {
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+  padding-top: 0.5rem;
+  color: #344563;
+}
+
+.modalFooter ul {
+  display: flex;
+  list-style: none;
+  gap: 8px;
+  color: #6070ff;
+  margin: 10px 24px 15px 0;
+}
+
+hr {
+  margin-bottom: 10px;
+}
+
+.modal-button {
```

Comment:
Kindly make sure your button matches with Figma design. Here, buttons have different sizes, icon and text is not aligned well and not enough between text and icon.👍
<img width="802" alt="Screenshot 2023-02-03 at 11 20 41 PM" src="https://user-images.githubusercontent.com/25887253/216707854-f8c08d57-ac3a-4b2d-8330-c84ad1eee4c3.png">
<img width="717" alt="Screenshot 2023-02-03 at 11 21 10 PM" src="https://user-images.githubusercontent.com/25887253/216707857-7c6a20c7-9a8e-4d3d-b623-cf471ffcd631.png">